client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/bootstrap/BootstrapManager.java

```diff
@@ -16,26 +16,26 @@
 
 package org.kaaproject.kaa.client.bootstrap;
 
-import java.util.List;
-
```

Comment:
I'm not against code style. I mean that code formatting is out of scope these changes. Code style can applied in scope of another specific ticket.